app/models/user.rb

```diff
@@ -256,6 +257,11 @@ def default_country_to_usa
       self.country ||= "USA"
     end
 
+    def sanitize_postal_code
+      return unless self.postal_code
+      self.postal_code = self.postal_code.delete(' ').upcase
```

Comment:
no need for self here

I didn't know about `delete`, good stuff, I think always did `tr(' ', '')`

Reply:
second `self` removed in 967e880. I'm not super crazy about the `delete`, as I don't find it intuitive that it removes all matches, but at least it's a word?